LogIn/UIExtensions.swift

```diff
@@ -5,23 +5,31 @@
 //  Created by Aleksei Permiakov on 07.12.2021.
 //
 
+
 import UIKit
 
 extension UITextField {
     func loginTextStyle() {
         self.backgroundColor = .systemGray6
```

Comment:
Also it make sense to make this function private and move to the login view

Reply:
make sense. Done